presto-docs/src/main/sphinx/release/release-0.201.rst

```diff
@@ -0,0 +1,41 @@
+=============
+Release 0.201
+=============
+
+General Changes
+---------------
+
+* Improve performance of queries that process structural types and contain joins, aggregations
+  or table writes.
+* Fix planning error when column names are reused in ORDER BY query.
+* Add support for ``current_user``.
+* Change grouped aggregations to use ``IS NOT DISTINCT`` semantics rather than equality semantics.
+* System memory pool is now unused by default and it will eventually be removed completely.
+  All memory allocations will now be served from the general/user memory pool. The old behavior
+  can be restored with the deprecated.legacy-system-pool-enabled config option.
+* Improve performance and memory usage for queries using :func:`row_number` followed by a
+  filter on the row numbers generated.
+* Improve performance and memory usage for queries using `ORDER BY` followed by a `LIMIT`
+
+SPI Changes
+-----------
+* Support resource group selection based on resource estimates.
+
+Security Changes
+----------------
+
+* Change rules in the :doc:`/security/built-in-system-access-control` for enforcing matches
+  between authentication credentials and a chosen username to allow more fine-grained
+  control and ability to define "superuser"-like credentials.
+
+Hive Changes
+------------
+
+* Replace ORC writer stripe minimum row configuration ``hive.orc.writer.stripe-min-rows``
+  with stripe minimum data size ``hive.orc.writer.stripe-min-size``.
+* Change ORC writer validation config ``hive.orc.writer.validate`` to switch to a
+  sampling percentage ``hive.orc.writer.validation-percentage``.  This can reduce the time
```

Comment:
The last sentence is strange. It is saying we can find corruption faster, or that this can reduce resource usage?